backend/hexa/user_management/models.py

```diff
@@ -192,10 +193,20 @@ class Meta:
     countries = CountryField(multiple=True, blank=True)
     url = models.URLField(blank=True)
     contact_info = models.TextField(blank=True)
+    logo = models.BinaryField(blank=True, null=True)
     members = models.ManyToManyField(User, through="OrganizationMembership")
 
     objects = OrganizationManager.from_queryset(OrganizationQuerySet)()
```

Comment:
I think this line should change to 
```python
    objects = DefaultSoftDeletedManager.from_queryset(OrganizationQuerySet)()
    all_objects = IncludeSoftDeletedManager.from_queryset(OrganizationQuerySet)()
```
to make sure that `Organization.objects.all()` filters out the deleted ones.